jvm/ml4ir-inference/src/test/integration/ml4ir/inference/tensorflow/TensorFlowInferenceIT.scala

```diff
@@ -23,7 +22,44 @@ class TensorFlowInferenceIT extends TestData {
   case class PredictionVector(sequence: Map[String, String], scores: Array[Float])
 
   /**
-   * Basic parsing of the Python CSV predicition.
+   * Class holding a Ranking sequence of documents from Python prediction.
+   */
+  case class StringMapQueryAndPredictions(queryContext: Map[String, String],
+                                          docs: List[Map[String, String]],
+                                          predictedScores: Array[Float])
+  object StringMapCSVLoader {
+
+    def loadDataFromCSV(dataPath: String, featureConfig: ModelFeaturesConfig): Iterable[StringMapQueryAndPredictions] = {
```

Comment:
@ducouloa Can you add docstrings for the arguments here?

Reply:
that's my bad - I didn't give @ducouloa my scaladocs for this:

`    /**
     *
     * @param dataPath fully qualified filesystem path to the "model_predictions.csv" file as produced by
     *                 the train_inference_evaluate mode of pipeline.py (see the README at
     *                 https://github.com/salesforce/ml4ir/tree/master/python )
     * @param featureConfig the in-memory representation of the "feature_config.yaml" that the python training 
     *                      process used for training
     * @return an iterable collection over what the training code got for ranking inference results, to compare with
     *         what the JVM inference sees
     */`

Reply:
Thanks a lot for this Jake.